src/KubeOps/KubeOps.csproj

```diff
@@ -21,6 +21,7 @@
             It enables a developer to create a custom controller for CRDs
             (CustomResourceDefinitions) that runs on kubernetes.
         </PackageDescription>
+        <GeneratePackageOnBuild>True</GeneratePackageOnBuild>
```

Comment:
That one is not needed ;-)

Reply:
gone now.  I turned that on for testing (reference the output directory as a nuget feed from another project), left behind accidentally when committing